src/axolotl/utils/models.py

```diff
@@ -548,6 +553,14 @@ def apply_patches(self) -> None:
 
             patch_self_attn_lora(self.cfg)
 
+        if self.cfg.sequence_parallel_degree > 1:
```

Comment:
```suggestion
        if self.cfg.sequence_parallel_degree and self.cfg.sequence_parallel_degree > 1:
```
should fix the NoneType comparison exception in the e2e tests